Earthfile

```diff
@@ -92,22 +92,45 @@ spec:
 # =============================================================================
 # Formatting
 # =============================================================================
-lint:
-    BUILD +lint-ocaml
-    BUILD +lint-python
+format:
```

Comment:
You probably have to update `+all` at the beginning of the file as well, to refer to `+format` instead of `+lint`.

Reply:
Indeed :facepalm: